apps/redi-connect/src/pages/app/me/stepper-card/HorizontalStepper.tsx

```diff
@@ -88,9 +112,17 @@ const HorizontalStepper = ({
             )}
           </>
         ) : (
-          <Content size="small" renderAs="p">
-            {steps[currentStep].message.mentee}
-          </Content>
+          <>
+            {isCyberspaceMentee ? (
+              <Content size="small" renderAs="p">
+                {steps[currentStep].message.menteeCyberspace}
+              </Content>
+            ) : (
+              <Content size="small" renderAs="p">
+                {steps[currentStep].message.mentee}
+              </Content>
+            )}
```

Comment:
I think the `mentee()` function idea is cleaner. Can we do a quick refactor?

Reply:
Yes, sure.